packages/dds/sequence/src/test/revertibles.spec.ts

```diff
@@ -174,6 +174,28 @@ describe("Sequence.Revertibles with Local Edits", () => {
 		collection.change(id, 3, 8);
 		collection.removeIntervalById(id);
 
+		revertIntervalRevertibles(sharedString, revertibles.splice(0));
+		assertIntervals(sharedString, collection, [{ start: 0, end: 5 }]);
+	});
+	it("checks id consistency when a remove is undone and redone", () => {
```

Comment:
Nit: this description is slightly misleading since a real "redo" would create a revertible from the add caused by the revert. In this case it's actually more like "checks revertibles still find correct interval across multiple remove and reverts"